docs/makefile

```diff
@@ -27,6 +27,12 @@ src/.gitbook/assets/%.svg: art/%.msc
 	@mkdir -p $(@D)
 	mscgen -T svg -i $< -o $@
 
+../target/debug/solana:
+	cd ../cli && cargo build
+
+src/cli/usage.md: ../target/debug/solana
```

Comment:
We'll want the script in the deps list as well

Reply:
Fixed in a15db7a, thanks